### PR TITLE
Add optimism to the nodes in the periodic endpoint

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: rotkehlchen/rotkehlchen.py [hunk 1]
```diff
@@ -1028,6 +1028,7 @@ def query_periodic_data(self) -> dict[str, Union[bool, list[str], Timestamp]]:
             with self.data.db.conn.read_ctx() as cursor:
                 result['last_balance_save'] = self.data.db.get_last_balance_save_time(cursor)
                 result['connected_eth_nodes'] = [node.name for node in self.chains_aggregator.ethereum.node_inquirer.get_connected_nodes()]  # noqa: E501
+                result['connected_optimism_nodes'] = [node.name for node in self.chains_aggregator.optimism.node_inquirer.get_connected_nodes()]  # noqa: E501
```
Comment (LefterisJP) [inline rotkehlchen/rotkehlchen.py:1031]: shouldn't this also have an api docs change? No other tests use this apart from the one you have the empty list in?

**Review by LefterisJP — APPROVED** [review]
LGTM. Docs l8er
